python/kfserving/kfserving/server.py

```diff
@@ -53,6 +53,8 @@ def __str__(self):
 KFSERVER_LOGLEVEL = os.environ.get('KFSERVER_LOGLEVEL', 'INFO').upper()
 logging.basicConfig(level=KFSERVER_LOGLEVEL)
 
+PREDICTOR_URL_FORMAT = "http://%s/v1/models/%s:predict"
```

Comment:
this seems to have the downside that the call has to go through the virtualservice to get the rewrite right? will this work if we decide we have a scenario where we want to place the transformer with the predictor in the same pod?

Reply:
No, this is internal mesh call which directly hit default/canary predictor service route from transformer or explainer, only external call goes through the `VirtualService` to get rewrite. If we place them on the same pod, they can communicate via localhost, the only difference I think is the host name.